website/assets/css/workshops.css

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.workshops-block {
+  border-left: 2pt solid lightgray;
+  position: relative;
+}
+
+.workshops-marker {
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  position: absolute;
+  left: calc(-0.5rem - 1pt);
+  background-color: lightgray;
+}
+
+.workshops-content {
+  margin-left: 1rem;
+  padding-bottom: 1rem;
+}
+
+.workshops-date {
+  color: gray;
+  padding-bottom: 0.25rem;
+}
+
+.workshops-table {
+  width: 70%;
```

Comment:
On mobile, the 70% width looks a bit cramped. I recommend either changing this to 100%, or adding the following to change the width to 100% on small screens: 

```cs
@media (max-width: 800px) {
  .workshops-table {
    width: 100%;
  }
}
```

Reply:
Changed to 100% width. Thanks for checking on mobile.